Paper.tex

```diff
@@ -1381,6 +1382,12 @@ \section{Transition to Proof of Stake} \label{ch:pos_transition}
 
 Upon reaching the terminal block, new blocks are processed by the Beacon Chain.
 
+\subsection{Post-Paris Updates}
+Because the Beacon Chain generate a new slot every 12 seconds, post-\textit{Paris} updates can be scheduled to occur at a specific timestamp.
+At the execution layer, the update will then happen in the first produced block after the scheduled timestamp.
+For example the \textit{Shanghai} hard fork was scheduled to occur at 2023-04-12 10:27:35 UTC, on Epoch 194,048.
+Validators failed to propose a block during the two first slots of this Epoch, but at slot 6,209,538 a validator finally proposed a the block 17,034,870, marking the transition to \textit{Shanghai} on the execution layer.
```

Comment:
proposed **a the** block

Reply:
changes in [2b98d68](https://github.com/ethereum/yellowpaper/pull/906/commits/2b98d68e5ea35e553fcc4fdab9c122ca91f55f38)